tensorflow_addons/optimizers/average_wrapper.py

```diff
@@ -38,7 +38,8 @@ def __init__(
             )
 
         self._optimizer = optimizer
-        self._track_trackable(self._optimizer, "awg_optimizer")
+        self._hyper.update(optimizer._hyper)
+        self._track_trackable(self._optimizer, "swg_optimizer")
```

Comment:
Emm, this is the `AverageOptimizerWrapper` class so what do swg and awg stand for?